src/sources/aws_s3/sqs.rs

```diff
@@ -555,11 +556,6 @@ impl IngestorProcess {
                     BatchStatus::Delivered => Ok(()),
                     BatchStatus::Errored => Err(ProcessingError::ErrorAcknowledgement),
                     BatchStatus::Rejected => {
-                        // We should emit `component_discarded_events_total` here, but is count always 1?
-                        error!(
-                            message = "Sink reported events were rejected.",
-                            internal_log_rate_secs = 10,
-                        );
                         // Failed events cannot be retried, so continue to delete the SQS source message.
```

Comment:
Should we expand on this comment to mention that we are intentionally not emitting error / events dropped here as it should be handled in the sink ?

Reply:
Probably, it's sort of similar to the note we add around the call to `encoding` that handles the error. I'm not sure if that's better suited for copied comments in each component or something more centralized.

Reply:
Although it isn't pretty, I am probably of the vote for copied comments in each component. Even a one liner that is enough of a hint to get the curious person to look for the more centralized and detailed comment.
But I defer to your judgement